flink-metrics/flink-metrics-slf4j/src/main/java/org/apache/flink/metrics/slf4j/Slf4jReporter.java

```diff
@@ -39,6 +41,8 @@
 /**
  * {@link MetricReporter} that exports {@link Metric Metrics} via SLF4J {@link Logger}.
  */
+@PublicEvolving
```

Comment:
This shouldn't be added, as it is an unrelated change.

Reply:
ok, i will resubmit after delete it

Reply:
cc zentol this change have commit,please check it ,thx